cmd/kubeadm/app/api/types.go

```diff
@@ -68,3 +72,11 @@ type ClusterInfo struct {
 	CertificateAuthorities []string `json:"certificateAuthorities"`
 	Endpoints              []string `json:"endpoints"`
 }
+
+// Master API endpoint connection
+type ConnectionDetails struct {
```

Comment:
@mikedanese recently put in a change that is aimed at transforming this to an API gorup. With that in mind, this really doesn't seem like an API type. Maybe there's a better place for it.